releases.md

```diff
@@ -37,6 +37,9 @@ updating OpenShift resources). These actions have to be described in
 * In `ods-project-quickstarters`, on the release branch, set all pointers to the
   shared library in each `Jenkinsfile` to the release branch (e.g. change the
   pointer from `production` to `1.2.x`).
+* In `ods-provisioning-app`, on the release branch, set the pointer to the
+  shared library in the `Jenkinsfile` to the release branch (e.g. change the
+  pointer from `production` to `1.2.x`).
```

Comment:
provide concrete example: e.g. 
`ods-library@0.1-latest` to `ods-library@1.0.0` in case the tag is called 1.0.0

Reply:
Not sure what you are referring to, there is a concrete example: `(e.g. change the pointer from `production` to `1.2.x`)`